packages/rocket-auth-aws-infrastructure/src/lambdas/response.ts

```diff
@@ -0,0 +1,23 @@
+import { APIGatewayProxyResult } from 'aws-lambda'
+import { AWSError } from 'aws-sdk'
+import { httpStatusCodeFor } from '@boostercloud/framework-types'
+
+export const response = (statusCode: number, data: object): APIGatewayProxyResult => {
+  return {
+    statusCode,
+    body: JSON.stringify(data),
+  }
+}
+
+export const errorResponse = (e: AWSError): APIGatewayProxyResult => {
+  return response(httpStatusCodeFor(e), {
+    error: {
+      type: e.code,
+      message: e.message,
+    },
+  })
+}
+
+export const okResponse = (data: object): APIGatewayProxyResult => {
```

Comment:
Is this function needed? Can't we call to `response(200, data)` directly?

Reply:
Its to give the same abstraction level

Imagine some code that goes like:

```ts
if (something) {
  errorResponse(...)
} else {
  response(200, ...)
}
```

It's okay, but the `else` branch is slightly low level than the `if` branch. So its good to hide the code implementation detail in order to make it slightly more abstract:

```ts
if (something) {
  errorResponse(...)
} else {
  okResponse(...)
}
```